### PR TITLE
fix(Button): remove loading placeholder from layout when not loading

### PR DESCRIPTION
**Pull request by MkMan**
This change removes the Loading component from layout to fix issues where it was messing up layout for siblings.

## Before
image

## After
image

### CODE REVIEW

File: src/components/Button/Button.module.css [hunk 1]
```diff
@@ -121,3 +121,8 @@
     }
   }
 }
+
+.hideLoadingIndicator {
```
Comment (cyrusxzw) [inline src/components/Button/Button.module.css:125]: How about using CSS Clipping?

 ```
.hideLoadingIndicator {
    border: 0;
    clip: rect(1px 1px 1px 1px); /* IE6, IE7 */
    clip; rect(1px, 1px, 1px, 1px);
    height: 1px;
    margin: -1px;
    overflow: hidden;
    padding: 0;
    position: absolute;
    width: 1px; 
} 
```

**Comment by cyrusxzw** [comment]
Good solution by adding a hidden className. Could you help me to check if it works well in carousel component without my fix?

File: src/components/Button/Button.module.css (hunk 1 above)
Comment (MkMan) [reply src/components/Button/Button.module.css:125]: I haven't tried it but I am keen to understand how it's better than `left: -999px`

Comment (cyrusxzw) [reply src/components/Button/Button.module.css:125]: I just feel like `left: -999px` is a bit of a hack. 😄

Comment (MkMan) [reply src/components/Button/Button.module.css:125]: Fair enough 👍  I'll see if this approach has an impact on a11y and give it a shot

Comment (cyrusxzw) [reply src/components/Button/Button.module.css:125]: My refs: https://www.nomensa.com/blog/how-improve-web-accessibility-hiding-elements

Comment (MkMan) [reply src/components/Button/Button.module.css:125]: I used the css-clip example shown in this [page](https://webaim.org/techniques/css/invisiblecontent/) which is referenced on the page you linked 😃 
Thanks for the suggestion

**Comment by MkMan** [comment]
> Good solution by adding a hidden className. Could you help me to check if it works well in carousel component without my fix?

Yes it works without the fix 👌 

image

image

Would you like me to revert it?

**Comment by cyrusxzw** [comment]
> > Good solution by adding a hidden className. Could you help me to check if it works well in carousel component without my fix?
> 
> Yes it works without the fix 👌
> 
> image
> 
> image
> 
> Would you like me to revert it?

Yes please, thanks!

**Review by cyrusxzw — APPROVED** [review]

**Pull request merged**